### PR TITLE
Process when Fabric is removed

### PR DESCRIPTION
**Pull request by jinwon0325kim**
When OnFabricRemoved is received from connectedhomeip, things are terminated.
When deleted from the fabric, a popup occurs as shown in the figure below.

popup

### CODE REVIEW

File: src/common/common_window.py [hunk 1]
```diff
@@ -324,7 +328,7 @@ def update_msg(self, str):  # received msg from ioterPipe.pipeThread
                 self.pushButtonDevicePower.toggle()
                 self.occur_abort.emit(self.device_info.com_port)
         else:
-            if self.pipe_event_handler is not None:
+            if self.pipe_event_handler and self.device_info.get_commissioning_state():
```
Comment (junyong-sim) [inline src/common/common_window.py:331]: Why did you add this condition ?

File: src/common/common_window.py [hunk 2]
```diff
@@ -272,6 +272,10 @@ def update_progress(self, step, msg):
         elif step == 80 and self.device_info.get_commissioning_state():
             self.stackedWidget.setCurrentIndex(1)
             self.progressBar.setValue(100)
+        elif step == 200:
+            re = QMessageBox.information(self, "Exit", "The fabric was removed.")
+            if re == QMessageBox.Ok:
```
Comment (junyong-sim) [inline src/common/common_window.py:277]: How about changing the text more easily ?
"The device was removed from App."

File: src/common/common_window.py (hunk 1 above)
Comment (jinwon0325kim) [reply src/common/common_window.py:331]: At the beginning of onboarding, the setting message appears as shown below.
So the setting message came out only when onboaring was completed.
[2023-06-19 18:12:31.463761][0] pair:20054912334
[2023-06-19 18:12:31.464135][0] qrcode:MT:6PNA20QV17550648G00
[2023-06-19 18:12:32.466952][0] threadsettingfile:0_f4ce36d53dcf5417.data
[2023-06-19 18:12:32.469515][0] onoff:0
[2023-06-19 18:12:32.470274][0] level:254
[2023-06-19 18:12:32.472265][0] colortemp:0
[2023-06-19 18:12:32.486752][0] step:1:Start Matter Commissioning

File: src/common/common_window.py (hunk 2 above)
Comment (jinwon0325kim) [reply src/common/common_window.py:277]: That's a good idea
I will modify the message according your opinion.

Comment (jinwon0325kim) [reply src/common/common_window.py:277]: I updated it

**Review by junyong-sim — APPROVED** [review]
LGTM
